bitsandbytes/__init__.py

```diff
@@ -24,4 +24,6 @@
     "optim.optimizer.MockArgs": False,
 }
 
+__version__ = "0.41.1"
```

Comment:
please update the version to the most recent one :)

Reply:
good call!

Reply:
... which changed yet again today )

Reply:
`0.41.3.post1` should be the current one though, correct? That's currently what we have in the PR here.